source/UIAHandler/__init__.py

```diff
@@ -414,19 +413,19 @@ def getUIAElementDebugString(self, element: UIA.IUIAutomationElement) -> str:
 		)
 
 	def __init__(self):
-		super(UIAHandler,self).__init__()
+		super(UIAHandler, self).__init__()
 		self.globalEventHandlerGroup = None
 		self.localEventHandlerGroup = None
 		self.localEventHandlerGroupWithTextChanges = None
 		self._localEventHandlerGroupElements = set()
-		self.MTAThreadInitEvent=threading.Event()
+		self.MTAThreadInitEvent = threading.Event()
 		self.MTAThreadQueue = Queue()
-		self.MTAThreadInitException=None
+		self.MTAThreadInitException = None

```

Comment:
please add typing here

Reply:
This is an exception, and a Stack Overflow post says we can't use typing to annotate exceptions (at this time) unless NVDA code base has found a way to do so which I may not be aware.